addons/web_editor/static/src/js/wysiwyg/widgets/link_tools.js

```diff
@@ -369,6 +369,15 @@ const LinkTools = Link.extend({
         if ($target.closest('[name="link_border_style"]').length) {
             return;
         }
+        if ($target.closest('[name="link_style_color"]')) {
+            // Reset custom styles when changing link style.
+            this.$link.css('color', '');
+            this.$link.css('background-color', '');
+            this.$link.css('background-image', '');
+            this.$link.css('border-width', '');
+            this.$link.css('border-style', '');
+            this.$link.css('border-color', '');
+        }
```

Comment:
@bso-odoo fyi, this should cover your remark.

Reply:
It works indeed, thank you !
Sorry for not grasping sooner what you meant...

Reply:
Don't worry about it!